front/app/api/project_folder_images/useAddProjectFolderImage.ts

```diff
@@ -0,0 +1,35 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import projectFolderImagesKeys from './keys';
+import { IProjectFolderImage, addProjectFolderImageObject } from './types';
+
+const addProjectFolderImage = async ({
+  folderId,
+  base64,
+}: addProjectFolderImageObject) =>
+  fetcher<IProjectFolderImage>({
+    path: `/project_folders/${folderId}/images`,
+    action: 'post',
+    body: { image: { image: base64 } },
```

Comment:
No need to nest it in this way either, we can just have `image: base64`

Reply:
Hmm I actually get an error from the BE if I use only `{ image: base64 }`. It looks like it's nested for some of the other addImage endpoints as well :thinking: 

Error:
![image](https://github.com/CitizenLabDotCo/citizenlab/assets/33987955/dcca9039-8f32-462c-9058-1042b24946b2)



E.g. Nested in add initiative image:
```
export interface AddInitiativeImageObject {
  initiativeId: string;
  image: {
    image: string;
  };
}
```